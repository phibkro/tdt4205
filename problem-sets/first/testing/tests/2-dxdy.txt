dx=2
dy=3
dx=9
dx=55
dy=09
dx=5 dx=1 go
da=1
indy=1
dx=999
dy=-10
dx=--5
dx=-0 dy=55 go dx=197 dy=-2
dx=20     dy=20
