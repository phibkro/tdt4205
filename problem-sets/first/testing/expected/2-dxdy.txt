Table filled!
line    1: accepted: dx=2
line    2: accepted: dy=3
line    3: accepted: dx=9
line    4: accepted: dx=55
line    5: accepted: dy=09
line    6: accepted: dx=5 dx=1 go
line    7: error: da=1
~~~~~~~~~~~~~~~~~~~^
line    8: error: indy=1
~~~~~~~~~~~~~~~~~~^
line    9: accepted: dx=999
line   10: accepted: dy=-10
line   11: error: dx=--5
~~~~~~~~~~~~~~~~~~~~~~^
line   12: accepted: dx=-0 dy=55 go dx=197 dy=-2
line   13: accepted: dx=20     dy=20
